src/routes/console/supportWizard.svelte

```diff
@@ -85,7 +102,10 @@
             If you found a bug or have questions, please reach out to the Appwrite team. We try to
             respond to all messages within our office hours.
         </p>
-        <p class="text u-margin-block-start-32">Available: <b>Mon-Fri 09:00 - 17:00 UTC</b></p>
+        <p class="text u-margin-block-start-32">
+            Available: <b>{supportWeekDays}, {supportTimings}</b>
+            <Label tooltip={localeTimezoneName()} />
```

Comment:
Let's just put the timezone in the paragraph. And can you also remove the illustration? So, it should look like:

![image](https://github.com/appwrite/console/assets/1477010/ef76ab86-73f8-47de-95c5-f21fe2963553)

Reply:
Done & Done.